solution/Makefile

```diff
@@ -161,20 +161,21 @@ python.unittest:
 python.shell:
 	docker-compose exec regulations python manage.py shell
 
+python.deletetables:
+	docker-compose exec regulations python manage.py cleantables
```

Comment:
Rename `python.deletetables` to cleantables for consistency. We're not deleting the table